aten/src/ATen/native/ReplicationPadding.cpp

```diff
@@ -216,6 +217,9 @@ Tensor& replication_pad1d_backward_out_cpu_template(
   /* get contiguous gradOutput */
   auto gradOutput = gradOutput_.contiguous();
   gradInput.resize_as_(input);
+  if (gradInput.numel() == 0) {
+    return gradInput;
```

Comment:
Should this return a clone instead? I think this is fine as is because it's a gradient (which shouldn't be operated on directly inplace), but looking at commit 29fabb1fbc and the discussion in gh-32612 I wanted to comment for another reviewer to make sure.

Reply:
Thanks for confirming.